qiita_pet/support_files/doc/source/checklist-for-ebi-ena-submission.rst

```diff
@@ -1,27 +1,29 @@
 Checklist to send data to EBI-ENA
 =================================
 
+`Here <https://knightlab.ucsd.edu/wordpress/wp-content/uploads/2016/04/QiitaTemplate_20181218.xlsx>`__ you will find a document outlining these requirements, with examples, when possible.
+
 For each preparation that needs to be uploaded to EBI-ENA we will check:
 
   1. Data processing
 
-    a. Make sure there is raw data linked - no BIOMs allowed
+    a. Only datasets where raw sequences are available and linked to the preparation can be submitted. Studies where the starting point is a BIOM table cannot be submitted, since EBI is a sequence archive
     b. The data is processed and the owner confirms the data is correct:
 
-      1. For target gene: data is demultiplexed (review split_library_log to make sure it seems fine) and there is at least a close reference (GG for 16S, Silva for 18S, UNITE for ITS) or trim/deblur artifacts. Trimming should be done with 90, 100 and 150 base pairs (prefered).
-      2. For shotgun: data is uploaded via per_sample_FASTQ and processed using Shogun/utree
+      1. For target gene: data is demultiplexed (review split_library_log to make sure each sample has roughly the expected number of sequences) and there is at least a close-reference (GG for 16S, Silva for 18S, UNITE for ITS) or trim/deblur artifacts. Trimming should be done with 90, 100 and 150 base pairs (preferred).
+      2. For shotgun: data is uploaded via per_sample_FASTQ and processed using Shogun/utree. Remember to remove sequencing data for any `human subject <https://www.hmpdacc.org/hmp/doc/HumanSequenceRemoval_SOP.pdf>`__
```

Comment:
I meant this link:
https://github.com/qiita-spots/qp-shogun/blob/master/notebooks/host_filtering.rst

But that other link would probably be handy as well.